pkg/controller/direct/logging/logmetric_controller.go

```diff
@@ -150,6 +150,24 @@ func (a *logMetricAdapter) Create(ctx context.Context, u *unstructured.Unstructu
 	if filter == "" {
 		return fmt.Errorf("filter is empty")
 	}
+	if a.desired.Spec.LoggingLogBucketRef != nil {
+		bucketName := a.desired.Spec.LoggingLogBucketRef.External
+		if bucketName != "" {
+			// todo acpana support resolving non external ref for loggingLogBucketRef
```

Comment:
super nit: suggest using the todo format, like `TODO(apana)`.

Reply:
I can start using this moving forward. It would be nice to have a linter for this hehe.